commands/exec.js

```diff
@@ -11,7 +11,7 @@ class Exec extends Command {
       category: 'System',
       extended: 'This will spawn a child process and execute the given command.',
       botPerms: ['SEND_MESSAGES'],
-      permLevel: 'Bot Owner'
+      permLevel: 'Police'
```

Comment:
Not completely sure if it's a wise idea to allow all police to use exec, but I'm not sure.

Reply:
I can restructure the perm levels, but then I'd need IDs of users that you'd like to be allowed to use exec.

Reply:
It would be a wise idea to leave it open to only the host of the bot and configurable.